addons/base_import_module_apps/views/ir_module_views.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record model="ir.ui.view" id="module_view_kanban_apps_inherit">
+            <field name="name">Apps Kanban Data Modules</field>
+            <field name="model">ir.module.module</field>
+            <field name="inherit_id" ref="base.module_view_kanban"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='application']" position="after">
+                    <field name="module_type"/>
+                </xpath>
+                <xpath expr="//button[@name='button_immediate_install']" position="after">
+                <button type="object" class="btn btn-primary btn-sm" name="button_immediate_install_app" attrs="{'invisible': [('state', '!=', 'app_store')]}" groups="base.group_system" context="{'module_name':name}">Activate</button>
```

Comment:
Maybe review the indentation (currently at the same level as the parent tag), and break the line into shorter parts?
```suggestion
                    <button type="object" class="btn btn-primary btn-sm" name="button_immediate_install_app"
                            attrs="{'invisible': [('state', '!=', 'app_store')]}" groups="base.group_system"
                            context="{'module_name':name}">
                        Activate
                    </button>
```